platforms/python/versions/bullseye/versionsToBuild.txt

```diff
@@ -6,6 +6,7 @@
 3.8.6, E3FF2839C048B25C084DEBE9B26995E310250568,
 3.8.12, E3FF2839C048B25C084DEBE9B26995E310250568,
 3.8.15, E3FF2839C048B25C084DEBE9B26995E310250568,
+3.8.16, E3FF2839C048B25C084DEBE9B26995E310250568,
```

Comment:
Curious why we are adding 3.8.16 back? Weren't we just going to release 3.11.1?

Reply:
3.8.16 was not causing issue for our parnters:
![image](https://user-images.githubusercontent.com/86327553/217602501-1693b42a-1809-4098-8615-70fd3bfb1a76.png)